_apps/auto-assign.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Auto Assign
+description: Add reviewers/assignees to pull requests when pull requests are opened.
+slug: auto-assign
+screenshots:
+- https://github.com/kentaro-m/auto-assign/raw/master/public/assets/screenshots/screenshot.png
+- https://github.com/kentaro-m/auto-assign/raw/master/public/assets/demo.gif
+authors:
+- kentaro-m
+repository: kentaro-m/auto-assign
+host: https://kentaro-m-auto-assign.glitch.me/
+---
+
+A GitHub App built with [Probot](https://github.com/probot/probot) that adds reviewers/assignees to pull requests when pull requests are opened.
```

Comment:
Can we nix 'A GitHub App built with [Probot](https://github.com/probot/probot)' since all the apps on the website are built with probot

Reply:
Fixed it. https://github.com/probot/probot.github.io/pull/228/commits/25677383a257164e4e578fa21791606292d6c414